src/coreclr/gc/gc.cpp

```diff
@@ -49691,7 +49715,6 @@ HRESULT GCHeap::Initialize()
 
             if (seg_mem == nullptr)
             {
-                GCToEEInterface::LogErrorToHost("STRESS_REGIONS couldn't allocate ro segment");
```

Comment:
Does it make sense to remove logs that we already had? I know this failure would be extremely rare, but I was trying to cover cases where the coreclr initialization could return E_FAIL and so users would have no idea where it came from and having the log at these places costs nothing.

Reply:
I see why you made the choices you made. why not log places that return other HRESULTs like E_OUTOMEMORY - those are actually more interesting because just telling someone there's OOM most likely doesn't help. 

my choices are for places that are likely hit by normal users and hard to debug currently. while it costs nothing - there are many, many places that return a non S_OK hr and I really didn't want to log in that many places so I just picked the ones I thought would be helpful. `STRESS_REGIONS` is not even defined normally - it's only used in private testing and in those cases it wouldn't be a problem for the person to look at where it fails.